src/components/tag/Tag.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+import { TAG_VARIANTS, DEFAULT_TAG_VARIANT, DEFAULT_TAG_SIZE, TAG_SIZES } from './Tag.constants';
+
+import Tag from './Tag';
+
+export const TagArgs = {
+    variant: DEFAULT_TAG_VARIANT,
+    size: DEFAULT_TAG_SIZE,
+    text: 'Lorem Ipsum Dolor',
+};
+
+export const TagArgTypes = {
+    variant: {
+        options: TAG_VARIANTS,
+        control: {
+            type: 'select',
+        },
+    },
+    size: { options: TAG_SIZES, control: { type: 'select' } },
+    onClick: { action: 'clicked' },
+};
+
+<Meta title={'Components/Tag'} />
+
+<ArgsTable of={Tag} />
+
+export const onClick = (variant) => {
+    if (variant === 'highlight') {
+        // eslint-disable-next-line no-console
+        console.log('#### Mention clicked!');
+    }
+};
```

Comment:
the `variant` can be taken from inside the component:

```suggestion
// eslint-disable-next-line no-console
export const onClick = (variant) => console.log('#### Mention clicked!');
```

But I think this was only meant for testing purposes, right?

Reply:
maybe I can remove it and leave it as an action control

Reply:
no ... keep in mind that the component should be generic regardless of our needs